tools/build_chain.sh

```diff
@@ -436,14 +438,14 @@ generate_config_ini()
 {
     local output=${1}
     local ip=${2}
-    local offset=$(get_value ${ip//./}_count)
+    local offset=$(get_value ${ip//./}_port_offset)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5318653)